### PR TITLE
Rearranged New/Update Episode form to better, more fluid layout.

### PR DESCRIPTION
**Pull request by DaveFIND**
Covers Tickets: 2171, 2172, 2173,2174, 2175

### CODE REVIEW

File: cdx_core/app/views/episodes/_form.haml [hunk 1]
```diff
@@ -4,10 +4,16 @@
   %script
     $(document).ready( function(){
     $('.panel-tab').hide();
-
+    if(is_new_entry){ $('.questions').hide(); }
```
Comment (tripledub) [inline cdx_core/app/views/episodes/_form.haml]: Indentation

Comment (DaveBlueFrontier) [inline cdx_core/app/views/episodes/_form.haml]: Haml doesn't allow indentation in script areas.  Mo has more work pending in this file so I will move the whole lot out to a JS file once he has written up the story for it.

File: cdx_core/app/controllers/patients_controller.rb [hunk 2]
```diff
@@ -22,12 +24,15 @@ def index
   end
 
   def show
-    @patient = Patient.find(params[:id])
     return unless authorize_resource(@patient, READ_PATIENT)
 
     @patient_json = Jbuilder.new { |json| @patient.as_json_card(json) }.attributes!
   end
 
+  def find_patient
```
Comment (tripledub) [inline cdx_core/app/controllers/patients_controller.rb]: Good refactor but should probably be a private method

Comment (DaveBlueFrontier) [inline cdx_core/app/controllers/patients_controller.rb]: Ah, missed that.  Will fix next time I look at the file.

**Pull request merged**